src/routes/auth/providers/apple.ts

```diff
@@ -3,12 +3,13 @@ import { Strategy, Profile } from '@nicokaiser/passport-apple'
 import { PROVIDERS } from '@shared/config'
 import { initProvider } from './utils'
 import { UserData } from '@shared/types'
+import { getGravatarUrl } from '@shared/helpers'
 
 const transformProfile = ({ id, name, email, photos }: Profile): UserData => ({
   id,
   email,
   display_name: name ? `${name.firstName} ${name.lastName}` : email,
-  avatar_url: photos?.[0].value
+  avatar_url: photos?.[0].value || email && getGravatarUrl(email)
```

Comment:
Are there any unexpected side-effects from this kind of logic? Could we have an instance where the `photos` param doesn't return a value but we don't want to overwrite with Gravatar?